std_msgs/Header header
float64[3] sensoresDistancia
